### ISSUE TITLE
TypeError: Cannot set property 'family' of undefined

### ISSUE THREAD

**Issue by tkriik** [iarna/rtf-parser#1]
Hi,

I tried to test this library on Node v6.9.4 and npm v3.10.10 with the following code snippet:

```
const fs = require('fs');
const rtf = require('rtf-parser');

var parser = rtf((err, doc) => {
    if (err) {
        console.log('error: %s', err);
    } else {
        console.log('doc: %s', doc);
    }
});

fs.createReadStream('sample.rtf').pipe(parser);
```

I get the following error:

```
/home/kriik/sketch/node_modules/rtf-parser/rtf-interpreter.js:222
    this.group.currentFont.family = 'roman'
                                  ^

TypeError: Cannot set property 'family' of undefined
    at RTFInterpreter.ctrl$froman (/home/kriik/sketch/node_modules/rtf-parser/rtf-interpreter.js:222:35)
    at RTFInterpreter.cmd$controlWord (/home/kriik/sketch/node_modules/rtf-parser/rtf-interpreter.js:98:19)
    at RTFInterpreter._write (/home/kriik/sketch/node_modules/rtf-parser/rtf-interpreter.js:59:19)
    at doWrite (/home/kriik/sketch/node_modules/rtf-parser/node_modules/readable-stream/lib/_stream_writable.js:347:64)
    at writeOrBuffer (/home/kriik/sketch/node_modules/rtf-parser/node_modules/readable-stream/lib/_stream_writable.js:336:5)
    at RTFInterpreter.Writable.write (/home/kriik/sketch/node_modules/rtf-parser/node_modules/readable-stream/lib/_stream_writable.js:274:11)
    at RTFParser.ondata (/home/kriik/sketch/node_modules/rtf-parser/node_modules/readable-stream/lib/_stream_readable.js:546:20)
    at emitOne (events.js:96:13)
    at RTFParser.emit (events.js:188:7)
    at readableAddChunk (/home/kriik/sketch/node_modules/rtf-parser/node_modules/readable-stream/lib/_stream_readable.js:217:18)
```

This is the `sample.rtf` being used as a test file: http://thewalter.net/stef/software/rtfx/sample.rtf

The same error occurs with the `rtf.string()` method as well when passing the contents of the file as a string. Am I doing something wrong, or is there an error in the library?

Thanks!

**Comment by k2s** [comment]
having the same problem. was able to partialy fix it in `rtf-interpreter.js`:

```
  ctrl$f (num) {
    if (this.group instanceof FontTable) {
      this.group.currentFont = this.group.table[num] = new Font()
    } else if (this.group.parent instanceof FontTable) {
      this.group.parent.currentFont = this.group.parent.table[num] = new Font()
      this.group.currentFont = this.group.parent.currentFont
    } else {
      this.group.style.font = num
    }
  }
```

but for your file I get now error: `Unsupported charset code #2`

**Comment by chuikoffru** [comment]
Такая же проблема.

/projects/jwbot/node_modules/rtf-parser/rtf-interpreter.js:252
    this.group.currentFont.charset = charset
                                   ^

TypeError: Cannot set property 'charset' of undefined

/projects/jwbot/node_modules/rtf-parser/rtf-interpreter.js:303
    this.currentFont.name = text.value.replace(/;\s*$/, '')
                          ^
TypeError: Cannot set property 'name' of null

**Comment by iarna** [comment]
Hey, thanks for reporting this, it should be fixed in 1.5
